src/libs/response-rules-interpreter.ts

```diff
@@ -13,7 +13,9 @@ import {
  * Get the first route response for which at least one rule is fulfilled.
  */
 export class ResponseRulesInterpreter {
-  private targets: { [key in Exclude<ResponseRuleTargets, 'header'>]: any };
+  private targets: {
+    [key in Exclude<ResponseRuleTargets, 'header' | 'body_plain'>]: any;
```

Comment:
Do we still need to exclude `body_plain`?
```suggestion
    [key in Exclude<ResponseRuleTargets, 'header'>]: any;
```

Reply:
Yup sorry, I forgot